docs/forcing.rst

```diff
@@ -1,5 +1,29 @@
 Forcing
 =======
+Data Override
+-------
+When running MOM6 with the Flexible Modelling System (FMS) coupler, forcing can be specified by a `data_table` file. This is particularly useful when running MOM6 with a data atmosphere, as paths to the relevent atmospheric forcing products (eg. JRA55-do or ERA5) can be provided here. Each item in the data table must be separated by a new line, and contains the following information:
+
+| ``gridname``: The component of the model this data applies to. eg. `atm` `ocn` `lnd` `ice`.
```

Comment:
Is this an example of the formatting for a yaml file? I am not too familiar with yaml formatting.

Reply:
Great thanks Theresa!

> Is this an example of the formatting for a yaml file? I am not too familiar with yaml formatting.

Admittedly I began writing out a yaml example, but I couldn't get the formatting right within the spninx documentation framework. It kept removing the indentations even when I was trying to write plain text blocks. I've also never seen a MOM6 repo that uses the yaml format so perhaps including only the legacy example in the docs is ok? 

Perhaps I could reword to mention that yaml format is also available, check the `data_override` docs for detail

Reply:
My last commit hopefully addresses this

Reply:
Thanks Ashley this is great!